deep-learning/_posts/2018-04-17-BCLearningClass.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: review
+title:  "Between-class Learning for Image Classification"
+tags:   deep-learning, CNN, classification
+author: "Charles Authier"
+pdf: "https://arxiv.org/pdf/1711.10284"
+cite:
+    authors: "Yuji Tokozume, Yoshitaka Ushiku, Tatsuya Harada"
+    title:   "Between-class Learning for Image Classification"
+    venue:   "https://arxiv.org/pdf/1711.10284"
+---
+
+## Intro
+In an other paper [^fn], they proposed a learning method named *Between-Class learning (BC learning)* for song recognition.They generated between-class examples by mixing two sounds belonging to different classes with a random ratio.They then input the mixed sound to the model and trained the model to output the mixing ratio of each class.
+
+The advantages of BC learning are not limited only to the increase in variation of the training data. They argued that BC learning has the ability to impose constraints on the feature distributions, which
+cannot be achieved with standard learning, and thus the generalization ability is improved.
+
+They achieved a performance surpasses the human level in sound classification tasks.
+
+This paper, is the awnser of the question: Can BC learning also perform on images?
```

Comment:
This paper tries to answer the question : does BC learning also applied to images?